pkg/panic-recovery/panic_recovery.go

```diff
@@ -3,11 +3,11 @@ package panic_recovery
 import (
 	"runtime"
 	"runtime/debug"
+	"slices"
 	"strings"
 
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
-	"golang.org/x/exp/slices"
```

Comment:
The `configuration/command.go` that I merged today also import this, so if you could replace that too that'd be nice!